connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -454,10 +448,9 @@ public GeyserSession(GeyserConnector connector, BedrockServerSession bedrockServ
         this.preferencesCache = new PreferencesCache(this);
         this.tagCache = new TagCache();
         this.worldCache = new WorldCache(this);
-        this.windowCache = new WindowCache(this);
+        this.formCache = new FormCache(this);
 
         this.collisionManager = new CollisionManager(this);
-
```

Comment:
Would keep this whitespace as these are two separate things being initialized. The code below the player entity initialization is part of the player entity section, in my opinion.